awesome-tray.el

```diff
@@ -1200,10 +1200,12 @@ If right is non nil, replace to the right"
   "Get new text to be displayed."
   (interactive)
   (let* ((tray-info (awesome-tray-build-active-info))
-         (minibuffer-info (current-message))
+         (minibuffer-info (or (current-message) ""))
+         (minibuffer-info (set-text-properties 0 (length minibuffer-info) nil minibuffer-info))
```

Comment:
Why set propertites here?

Reply:
Some minibuffer-info have properties, which causes problems with `split-string`. Need to delete all properties.